test/PushNotificationTests/APITests.cpp

```diff
@@ -173,6 +173,15 @@ namespace Test::PushNotifications
             VERIFY_NO_THROW(LocalBackgroundTask.Run(mockBackgroundTaskInstance));
         }
 
+        TEST_METHOD(BackgroundActivation_Unpackaged)
+        {
+            RunTestUnpackaged(L"BackgroundActivationTest", testWaitTime()); // Need to launch one time to enable background activation.
+
+            auto LocalBackgroundTask = winrt::create_instance<winrt::Windows::ApplicationModel::Background::IBackgroundTask>(c_comServerId, CLSCTX_ALL);
+            auto mockBackgroundTaskInstance = winrt::make<MockBackgroundTaskInstance>();
+            VERIFY_NO_THROW(LocalBackgroundTask.Run(mockBackgroundTaskInstance));
```

Comment:
Similar comment as in `MultipleBackgroundActivation_Unpackaged` test.

Reply:
Similar answer :)